products/zoningtaxlots/sql/bbl.sql

```diff
@@ -1,19 +1,20 @@
 -- insert unique bbls into table
 INSERT INTO dcp_zoning_taxlot (
+    dtm_id,
     bbl,
     boroughcode,
     taxblock,
     taxlot,
     area
 )
 SELECT
+    id,
     bbl,
     boro,
     block,
     lot,
-    ST_AREA(ST_MULTI(ST_UNION(geom))::geography) AS area
-FROM dof_dtm
-GROUP BY bbl, boro, block, lot;
```

Comment:
There was already a group by happening [here](https://github.com/NYCPlanning/data-engineering/pull/280/files#diff-3c953c8ca77cda9b1441cdcf641015b5bcd861e4680648f93b04c294a0b9c629R46) so cleaned this up